samples/MobileBuyIntegration/app/src/main/java/com/shopify/checkout_sdk_mobile_buy_integration_sample/common/MobileBuyEventProcessor.kt

```diff
@@ -82,20 +82,13 @@ class MobileBuyEventProcessor(
         logger.log("Checkout canceled")
     }
 
-    override fun onPermissionRequest(permissionRequest: PermissionRequest) {
-        logger.log("Permission requested for ${permissionRequest.resources}")
-        context.getActivity()?.let { activity ->
-            if (Permissions.hasPermission(activity, permissionRequest)) {
-                permissionRequest.grant(permissionRequest.resources)
-            } else {
-                ActivityCompat.requestPermissions(
-                    activity,
-                    arrayOf(Manifest.permission.CAMERA, Manifest.permission.RECORD_AUDIO),
-                    Permissions.PERMISSION_REQUEST_CODE,
-                )
-                permissionRequest.deny()
-            }
-        }
+    override fun onShowFileChooser(
```

Comment:
I'm replacing the onPermissionRequest() implementation, as it's not needed for the flow we have a use-case for, which doesn't launch a camera feed directly, but instead needs to show a file chooser.